tests/pubSub.cpp

```diff
@@ -205,6 +206,60 @@ BOOST_AUTO_TEST_CASE(test_publish_blocking_receive_zeroconf)
     BOOST_CHECK( publisher.join( ));
 }
 
+void onLargeEcho( const zeq::Event& event )
+{
+    BOOST_CHECK( event.getType() == zeq::vocabulary::EVENT_ECHO );
+}
+
+BOOST_AUTO_TEST_CASE(test_publish_receive_filters)
+{
+    zeq::Publisher publisher( lunchbox::URI( "foo://" ), zeq::ANNOUNCE_NONE );
+    zeq::Subscriber subscriber( publisher.getURI( ));
+    const std::string message( 60000, 'a' );
+
+    // Add unused handler to enable filtering
+    BOOST_CHECK( subscriber.registerHandler( EVENT_EXIT,
+                       std::bind( &test::onExitEvent, std::placeholders::_1 )));
+
+    // Make sure we're connected
+    BOOST_CHECK( subscriber.registerHandler( EVENT_ECHO,
+                       std::bind( &test::onEchoEvent, std::placeholders::_1 )));
+    for( size_t i = 0; i < 20; ++i )
+    {
+        BOOST_CHECK( publisher.publish(
+                         zeq::vocabulary::serializeEcho( test::echoMessage )));
+        if( subscriber.receive( 100 ))
+            break;
+    }
+    BOOST_CHECK( subscriber.deregisterHandler( EVENT_ECHO ));
+
+    // benchmark with no data to be transmitted
+    const zeq::Event& event = serializeEcho( message );
+    lunchbox::Clock clock;
```

Comment:
I know Lunchbox is a dependency but can not we use boost counterparts for measuring time.
for instance: https://kuniganotas.wordpress.com/2011/01/14/measuring-time-with-boost-library/